src/pybind/cephfs/cephfs.pyx

```diff
@@ -972,7 +975,10 @@ cdef class LibCephFS(object):
         if not isinstance(flags, int):
             raise TypeError('flags must be a int')
         if not isinstance(value, bytes):
-            raise TypeError('value must be a bytes')
+            if isinstance(value, str):
```

Comment:
the expected type of `value` is `byte`, not `str`.

Reply:
@tchaikov wouldn't it better to keep the conversion in the function definition rather than doing it every time before the function call?

Reply:
could you add a helper method in `CephFSVolumeClient` ?

Reply:
@tchaikov  Yeah. That would be a better approach.